addons/point_of_sale/models/pos_session.py

```diff
@@ -126,12 +126,20 @@ def _compute_cash_all(self):
     @api.constrains('user_id', 'state')
     def _check_unicity(self):
         # open if there is no session in 'opening_control', 'opened', 'closing_control' for one user
-        if self.search_count([('state', 'not in', ('closed', 'closing_control')), ('user_id', '=', self.user_id.id)]) > 1:
+        if self.search_count([
+                ('state', 'not in', ('closed', 'closing_control')),
+                ('user_id', '=', self.user_id.id),
+                ('name', 'not like', 'RESCUE FOR'),
```

Comment:
On second thought, I'd rather not use `=like` at all anywhere, it makes the domains harder to read. And if someone really decides to use "RESCUE FOR" as the prefix in their POS sequence, I'd say they asked for trouble and are likely to get some anyhow.